contracts/chainlink/AnteChainlinkHeartbeatTest.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import "../interfaces/IERC20.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorInterface.sol";
+import "hardhat/console.sol";
```

Comment:
Looks like `hardhat/console.sol` isn't used for the final version of this test, so safe to remove

Reply:
Removed it